SF iOS/SF iOS/SwipableNavigationContainer.m

```diff
@@ -100,15 +96,21 @@ - (UIViewController *)pageViewController:(UIPageViewController *)pageViewControl
 }
 
 - (void)controller:(nonnull GroupCollectionViewController *)controller tappedGroup:(nonnull Group *)group {
-     [self.mainNav pushViewController:[self viewControllerForGroup:group]
-                            animated:true];
+	[self.mainNav presentViewController:[self viewControllerForGroup:group] animated:YES completion:nil];
 }
 
 - (UIViewController *)viewControllerForGroup:(Group *)group {
     UITableView *tableView = [[EventsFeedTableView alloc] initWithFrame:CGRectZero style:UITableViewStylePlain];
     EventDataSource *dataSource = [[EventDataSource alloc] initWithGroup:group];
     [dataSource refresh];
-    return [[EventsFeedViewController alloc] initWithDataSource:dataSource
-                                                      tableView:tableView];
+    EventsFeedViewController *eventsFeedViewController = [[EventsFeedViewController alloc] initWithDataSource:dataSource
+																									tableView:tableView];
+	eventsFeedViewController.navigationItem.leftBarButtonItem = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemDone target:self action:@selector(dismiss:)];
+	return [[UINavigationController alloc] initWithRootViewController:eventsFeedViewController];
```

Comment:
Overall I like the PR but I'd like to change to this:

```
	UINavigationController *nav = [[UINavigationController alloc] initWithRootViewController:eventsFeedViewController];
    nav.navigationBar.prefersLargeTitles = true;
    return nav;
```

I really like the large titles.